platform/android/os_android.cpp

```diff
@@ -286,6 +300,10 @@ void OS_Android::vibrate_handheld(int p_duration_ms) {
 	godot_java->vibrate(p_duration_ms);
 }
 
+String OS_Android::get_config_path() const {
+	return "/sdcard/godot";
```

Comment:
need a better way of deciding that

Reply:
Addressed in https://github.com/thebestnom/godot/pull/2